keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/LifecycleEventRepositoryTests.kt

```diff
@@ -131,5 +132,27 @@ abstract class LifecycleEventRepositoryTests<T: LifecycleEventRepository> : JUni
         }
       }
     }
```

Comment:
This is a related test, but wouldn't have caught the problem But I added it because it seems worth it to test.